### DISCUSSION TITLE
그 저희 organization이 이름이 투박해서 그런데 다른 걸로 바꾸면 어떨까요?

### DISCUSSION THREAD

**Discussion by chadseok** [wanted-pre-onboarding-side-project/habit-tracker#44] [Ideas]
갑자기 생각난 거라 아직 저도 마땅히 생각나는 건 없는데 wanted-pre-onboarding-side-project는 좀 투박한 것 같아서요! ㅎㅎ

**Comment by jiyeon2** [comment]
맞아요 이름 생각이 안나서 아무거나 쓴거라 너무 밋밋하고 의미도 없어요..
저도 떠오르는 게 없는데
생각나는 게 있으면 여기에 올릴게요!!

**Reply by greyHairChooseLife** [reply]
저도.. 이거 막 생성 될 당시에 원티드?? 하긴 했는데 마땅히 제시할 대안이 안떠올라서 가만 있었습니다 ㅋㅋ  적절하게 바꾸면 좋을 것 같아요.
